pages/common/join.md

```diff
@@ -18,3 +18,7 @@
 - Produce a line for each unpairable line for file1:
 
 `join -a {{1}} {{file1}} {{file2}}`
+
+- Join two files, the '-' file from standard input:
+
+`join {{-}} {{file2}}`
```

Comment:
This is an old page, now we use `path/to/file`, also use `cat` to explain the stdin stuff:
```suggestion
`cat {{path/to/file1}} | join - {{path/to/file2}}`
```

Reply:
I'm happy to know it, thank you